app/models/user.rb

```diff
@@ -10,17 +10,17 @@ class User < ApplicationRecord
 
   validates_presence_of :email
   validates_uniqueness_of :email
-  validates_format_of :email, with: /.+@.+\..+/i
-  attr_accessor :skip_user_id_assign
-  before_save :assign_user_id, on: :create
+  validates_format_of :email, :with => /.+@.+\..+/i
```

Comment:
Another rubocop warning

I'm wondering if this is missing some of your recent mods?